### ISSUE TITLE
Add '--raise_error_at_min_scale False' flag in pretrain and finetune scripts to avoid  minimum loss scale crash caused by bad batch.

### ISSUE THREAD

**Issue by RobotiX101** [TinyLLaVA/TinyLLaVA_Factory#107]
To save your day, if any one encounter crash in finetune/pretrain and find this log:

```
exception: current loss scale already at minimum - cannot decrease scale anymore. exiting run
```

add ''--raise_error_at_min_scale False' flag in finetune or pretrain scripts.
